### ISSUE TITLE
Can't plant any sappling on TFC soils

### ISSUE THREAD

**Issue by Royal-Jelly** [supermassimo/DynamicTrees-ExC#42]
Other Dynamic Trees addons seem to work :100: percent fine though.

**Comment by supermassimo** [comment]
Moved to https://github.com/Gaelmare/dynamictreestfc/issues/27

**Comment by Royal-Jelly** [comment]
sorry again about the misunderstanding,

Please consider reopening and looking at this, and thank you kindly if you are able to help any.

**Comment by supermassimo** [comment]
If you could please elaborate on what the issue you are experiencing is exactly it would help quite a lot.
 Which trees, which soils, which addons you have installed and what versions of them. Which you have tried and work, which you have tried and didn’t, etc.

**Comment by Royal-Jelly** [comment]
**Tested again with only Terrafirmacraft and the following mods:**

**Dynamic trees - Quark [version - 1.0.4 ]** 
- blossoming seed - Plants on Terrafirmacraft soils


**Dynamic trees - Totemic [version - 1.0.1 ]** 
- red cedar cone - Does not plant on Terrafirmacraft soils


**Dynamic trees - The Twilight forest [version - 1.1.4 ]**
- rainbow twilight oak acorn - Does not plant on Terrafirmacraft soils
- sickly twilight oak acorn - Does not plant on Terrafirmacraft soils
- robust twilight oak acorn - Does not plant on Terrafirmacraft soils
- canopy seed - Does not plant on Terrafirmacraft soils
- mangrove seed - Does not plant on Terrafirmacraft soils
- darkwood seed - Does not plant on Terrafirmacraft soils
- sorting seed - Does not plant on Terrafirmacraft soils
- miner's seed - Does not plant on Terrafirmacraft soils
- seed of transformation - Does not plant on Terrafirmacraft soils
- seed of time - Does not plant on Terrafirmacraft soils

**Comment by Royal-Jelly** [comment]
I have not tested all of your other addons . . .

**Comment by Harleyoc1** [comment]
Since they seem to be mod-dependent I would expect it would be something to do with the order of the mods loading.

**Comment by supermassimo** [comment]
DT-TFC should then add the soils at the end of the loading sequence to ensure all trees are registered.
Best place would be CommonProxy postInit.
Here is the code i use, to make sure only trees that allow grass get (in this case, Aether Grass)

```java
	public void postInit() {
		for (Species species : Species.REGISTRY){
			if (species.isAcceptableSoil(null, null, Blocks.GRASS.getDefaultState())){ //if normal grass is allowed, then aether grass and dirt should be too
				species.addAcceptableSoil(BlocksAether.aether_grass, BlocksAether.enchanted_aether_grass, BlocksAether.aether_dirt);
			}
		}
	}
```

**Comment by supermassimo** [comment]
Fixed in [0.9.13](https://www.curseforge.com/minecraft/mc-mods/dynamictreestfc/files/3060333)
